examples/esp-chat-react/js/components/ChatApp.react.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var MessageSection = require('./MessageSection.react');
+var React = require('react');
+var ThreadSection = require('./ThreadSection.react');
+
+var ChatApp = React.createClass({
+
+    componentWillMount: function () {
+        this.props.router
+            .getModelObservable(this.props.modelId)
+            .observe(function (model) {
+                this.setState({ model: model });
```

Comment:
Done. Just added it for the singleton components, `threadSection` and `messageSection`.